.circleci/config.yml

```diff
@@ -19,25 +19,25 @@ jobs:
 
   publish_image_to_quay:
     docker:
-      - image: circleci/openjdk:8-jdk-stretch
+      - image: docker.io/docker:20.10-git
 
     steps:
       - checkout
       - setup_remote_docker:
-          version: 18.06.0-ce
+          version: 19.03.13
       - run:
           command: docker run --privileged linuxkit/binfmt:v0.7
       - run: |
-          docker build -t quay.io/prometheus/cloudwatch-exporter:latest .
+          docker build -t quay.io/prometheus/cloudwatch-exporter:latest --pull .
           if [[ -n "$CIRCLE_TAG" ]]; then
             docker tag -t quay.io/prometheus/cloudwatch-exporter:"${CIRCLE_TAG}" quay.io/prometheus/cloudwatch-exporter:latest
           fi
       - run: docker images
       - run: docker login -u $QUAY_LOGIN -p $QUAY_PASSWORD quay.io
       - run: |
-          docker push quay.io/prometheus/cloudwatch-exporter:latest
+          docker push --quiet quay.io/prometheus/cloudwatch-exporter:latest
```

Comment:
I don't think hiding output makes sense here.

Reply:
On CircleCI the carriage returns for the push progress don't work so you can end up with a lot of log spam. I'll drop the commit.